kalite/distributed/management/commands/setup.py

```diff
@@ -423,20 +422,23 @@ def handle(self, *args, **options):
                 print(
                     "If you have already downloaded the assessment items package, you can specify the file in the next step.")
                 print("Otherwise, we will download it from {url}.".format(
-                    url=ASSESSMENT_ITEMS_ZIP_URL))
+                    url=CONTENTPACK_URL))
 
-                if raw_input_yn("Do you wish to download the assessment items package now?"):
-                    ass_item_filename = ASSESSMENT_ITEMS_ZIP_URL
-                elif raw_input_yn("Have you already downloaded the assessment items package?"):
+                if raw_input_yn("Do you wish to download the content pack now?"):
+                    ass_item_filename = CONTENTPACK_URL
+                    retrieval_method = "download"
+                elif raw_input_yn("Have you already downloaded the content pack?"):
                     ass_item_filename = get_assessment_items_filename()
+                    retrieval_method = "local"
                 else:
                     ass_item_filename = None
+                    retrieval_method = "local"
 
                 if not ass_item_filename:
                     logging.warning(
-                        "No assessment items package file given. You will need to download and unpack it later.")
+                        "No content pack given. You will need to download and unpack it later.")
                 else:
-                    call_command("unpack_assessment_zip", ass_item_filename)
+                    call_command("retrievecontentpack", retrieval_method, "en", ass_item_filename)
```

Comment:
So if I had been using `unpack_assessment_zip` I should replace it with an equivalent `retrievecontentpack`?